packages/plugin-git/sources/gitUtils.ts

```diff
@@ -0,0 +1,32 @@
+import {execUtils}                                    from '@berry/core';
+import {xfs, NodeFS, PortablePath, ppath, Filename}   from '@berry/fslib';
+
+const tmp = require('tmp');
+
+export async function clone(repo: string) {
+  const directory = await createTemporaryFolder()
+  const {code, stdout} = await execUtils.execvp(`git`, [`clone`, `${repo}`, `${directory}`], {cwd: directory});
```

Comment:
Would be good to add support for the [`enableNetwork` setting](https://github.com/yarnpkg/berry/blob/master/packages/berry-core/sources/Configuration.ts#L238) 🤔

Ref: https://stackoverflow.com/a/41369667/880703

Reply:
Oh and the Batch Mode must be enabled, to prevent SSH asking for data and causing installs to hang when the SSH keys are missing.

Ref: https://unix.stackexchange.com/a/264477/24106

Reply:
Implemented the requested changes 💪